imports/services/isEditedService.js

```diff
@@ -0,0 +1,279 @@
+import { Meteor } from 'meteor/meteor';
+import '/imports/helpers/promisedMethods';
+
+import { MinutesSchema } from '../collections/minutes.schema';
+import { Minutes } from '../minutes';
+import { MeetingSeriesSchema } from '../collections/meetingseries.schema';
+import { MeetingSeries } from '../meetingseries';
+import {Topic} from '../topic';
+
+function setIsEditedMS(msId) {
+    let ms = new MeetingSeries(msId);
+
+    ms.isEditedBy = Meteor.userId();
+    ms.isEditedDate = new Date();
+
+    ms.save();
+}
+
+function removeIsEditedMS(msId, any) {
+    let unset = false;
+    let ms = new MeetingSeries(msId);
+
+    if (any === true) {
+        unset = true;
+    }
+    else {
+        if (ms.isEditedBy === Meteor.userId()) {
+            unset = true;
+        }
+    }
+
+    if (unset === true) {
```

Comment:
Same here

Reply:
Could you precise what i should improve?

Reply:
This is related to https://github.com/4minitz/4minitz/pull/355#discussion_r146111701

You don't need to explicitly check for `unset === true`. Checking `if (unset)` is sufficient.